### PR TITLE
Switch from Bodymovers to Constraints

### PR DESCRIPTION
**Pull request by xvarmkorv2**

### CODE REVIEW

**Comment by P3tray** [comment]
Oh hey! Deja vu!

Time for this PR to be left open for the next 16 months.

File: MainModule/Server/Dependencies/Assets/Spinner.client.lua [hunk 1]
```diff
@@ -1,6 +1,6 @@
 local torso = script.Parent
-local bg = torso:FindFirstChild("SPINNER_GYRO")
+local bg = torso:FindFirstChild("ADONIS_SPIN_GYRO")
 repeat
-  wait(1/44)
-  bg.cframe = bg.cframe * CFrame.Angles(0,math.rad(30),0)
+  task.wait(1/44)
```
Comment (P3tray) [inline MainModule/Server/Dependencies/Assets/Spinner.client.lua:4]: just make it 0.0227 like a normal person

**Review by P3tray — CHANGES_REQUESTED** [review]
looks good nothing really to scrutinize so approved (⌐■_■)

**Review by P3tray — APPROVED** [review]
👍🏿

**Comment by DimensionalFusion** [comment]
> 👍🏿

Ew

**Pull request merged**